app/models/katello/host/content_facet.rb

```diff
@@ -257,16 +343,16 @@ def self.inherited_attributes(hostgroup, facet_attributes)
         property :upgradable_rpm_count, Integer, desc: 'Returns upgradable RPM count'
         property :content_source, 'SmartProxy', desc: 'Returns Smart Proxy object as the content source'
         prop_group :katello_idname_props, Katello::Model, meta: { resource: 'content_source' }
-        prop_group :katello_idname_props, Katello::Model, meta: { resource: 'content_view' }
+        # prop_group :katello_idname_props, Katello::Model, meta: { resource: 'content_view' }
         property :errata_counts, Hash, desc: 'Returns key=value object with errata counts, e.g. {security: 0, bugfix: 0, enhancement: 0, total: 0}'
         property :kickstart_repository, 'Repository', desc: 'Returns Kickstart repository object'
         prop_group :katello_idname_props, Katello::Model, meta: { resource: 'kickstart_repository' }
-        prop_group :katello_idname_props, Katello::Model, meta: { resource: 'lifecycle_environment' }
+        # prop_group :katello_idname_props, Katello::Model, meta: { resource: 'lifecycle_environment' }
```

Comment:
I'm not sure what to do with these (or how they work..)

Reply:
ahh, okay! Looks like it adds a `content_view_id` and `lifecycle_environment_id` to the docs.

I think it's correct to just remove these, then, since `content_view_environments` isn't actually an attribute of content facets.